draft-ietf-privacypass-protocol.md

```diff
@@ -103,6 +103,44 @@ Unless otherwise specified, this document encodes protocol messages in TLS
 notation from {{!TLS13=RFC8446}}, Section 3. Moreover, all constants are in
 network byte order.
 
+# Protocol Overview
+
+The issuance protocols defined in this document emody the core of Privacy Pass.
+Clients receive TokenChallenge inputs from the redemption protocol
+({{AUTHSCHEME, Section 2.1}}) and use the issuance protocols to produce
+corresponding Token values ({{AUTHSCHEME, Section 2.2}}). The issuance protocol
+describes how Clients and Issuers interact to compute a token using a one-round
+protocol consisting of a TokenRequest from the Client and TokenResponse from
+the Issuer. This interaction is shown below.
+
+~~~ aasvg
+  Origin             Client        Attester          Issuer
+
+                    +--------------------------------------\
```

Comment:
Do we want the fancy stuff Martin added?
```+-----------------------------------.```, etc?

Reply:
Oh, I must have missed that. Let me apply it. Stand by.

Reply:
Fixed